docs/system/k8s-metrics.md

```diff
@@ -19,6 +19,8 @@ and therefore inherit its attributes, like `k8s.pod.name` and `k8s.pod.uid`.
 
 - [Pod metrics](#pod-metrics)
   - [Metric: `k8s.pod.uptime`](#metric-k8spoduptime)
+  - [Metric: `k8s.pod.phase`](#metric-k8spodphase)
```

Comment:
Not sure if there has been a discussion on this already, but out of curiosity, is there any reason not to use `k8s.pod.status.phase`? Both of these metrics refer to adjacent fields in the same PodStatus v1 core endpoint, but their prefixes differ in this convention.

In other metrics, when would we use the `k8s.pod.status` prefix vs just `k8s.pod`?

Coming from a Prometheus world, the equivalent kube-state-metrics are consistently named as:

- `kube_pod_status_phase`
- `kube_pod_status_reason`

Hence why I ask the question :)

I would suggest that both of the metrics be consistently named with each other, either:

- `k8s.pod.phase`
- `k8s.pod.reason`

or

- `k8s.pod.status.phase`
- `k8s.pod.status.reason`

Reply:
Using `k8s.pod.status.phase` for consistency makes sense. Thank's for the suggestion!
I've changed that with e35f06f.